python/astro_metadata_translator/translators/decam.py

```diff
@@ -314,6 +314,12 @@ def to_detector_name(self) -> str:
         name = self.to_detector_unique_name()
         return name[1:]
 
+    @cache_translation
+    def to_focus_z(self) -> u.Quantity:
+        # Docstring will be inherited. Property defined in properties.py
+        tel_focus_list = self._header["TELFOCUS"].split(",")
```

Comment:
Probably could do with a comment here explaining the content of TELFOCUS is a comma-separated string with six values.